lib/copy.go

```diff
@@ -25,6 +26,15 @@ import (
 // Copy will copy the directory/file at from to the path to inside the untarred
 // ACI at acipath.
 func Copy(acipath, from, to string) error {
```

Comment:
Can we move to this? https://github.com/coreos/rkt/blob/master/pkg/fileutil/fileutil.go#L63
I'm happy for the interface to change slightly if it makes it easier, I think the uid stuff might be a bit of a leak where a generic callback would be better.